lib/internal/fs/watch/linux.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+const {
+  SafeMap,
+  StringPrototypeStartsWith,
+  Symbol,
+  SymbolAsyncIterator,
+} = primordials;
+
+const {
+  codes: {
+    ERR_FEATURE_UNAVAILABLE_ON_PLATFORM,
+    ERR_INVALID_ARG_VALUE,
+  },
+} = require('internal/errors');
+const { getValidatedPath } = require('internal/fs/utils');
+const { kEmptyObject } = require('internal/util');
+const { validateObject, validateBoolean, validateAbortSignal } = require('internal/validators');
+const { EventEmitter, once } = require('events');
+const path = require('path');
+
+const kFSWatchStart = Symbol('kFSWatchStart');
```

Comment:
I think we should use the same symbol as the native implementation:
```suggestion
const { kFSWatchStart } = require('internal/fs/watchers');
```